opentelemetry-api/setup.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import find_packages
+from setuptools import setup
+from opencensus.ext.azure.common.version import __version__
+
+setup(
+    name='opentelemetry-api',
+    version=__version__,  # noqa
+    author='OpenTelemetry Authors',
+    author_email='cncf-opentelemetry-contributors@lists.cncf.io',
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache Software License',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+    ],
+    description='OpenTelemetry Python API',
+    include_package_data=True,
+    long_description=open('README.rst').read(),
+    install_requires=[
+    ],
```

Comment:
Nit: lose the newline

Reply:
I put this way as we will add the contextvars backport dependency, which will be one line change instead of three.